strider/storage.py

```diff
@@ -0,0 +1,99 @@
+"""Storage."""
+from abc import ABC
+import os
+import json
+import logging
+
+host = os.getenv('REDIS_HOST', 'redis')
+
+if host == 'fakeredis':
+    import fakeredis
+    r = fakeredis.FakeRedis(
+        encoding="utf-8",
+        decode_responses=True,
+    )
+else:
+    import redis
```

Comment:
I'm a fan of aioredis, but we couldn't async-ify logging without ditching the standard library, which I don't want to do.